src/p2p/net_node.inl

```diff
@@ -1243,8 +1243,8 @@ namespace nodetool
     if(!max_index)
       return 0;
 
-    size_t x = crypto::rand<size_t>()%(max_index+1);
-    size_t res = (x*x*x)/(max_index*max_index); //parabola \/
+    size_t x = crypto::rand<size_t>()%(16*max_index+1);
```

Comment:
I'd:
1) use const for the the 2 variables.
2) use a const or constexpr for the 16 and reuse it.